flink-core/src/main/java/org/apache/flink/types/parser/ByteParser.java

```diff
@@ -21,22 +21,23 @@
 
 
 public class ByteParser extends FieldParser<Byte> {
-	
```

Comment:
Please avoid such reformatting changes in the future.
In this file less than 10 changed lines make up the actual change but almost 80 lines are touched. This costs quite a bit of time to review.